lib/modules/manager/pixi/readme.md

```diff
@@ -1,5 +1,35 @@
-Limited support for pixi package manager
+### Pixi package manager
 
-Whenever the pixi config in `pyproject.toml` or `pixi.toml` file is updated, the `pixi.lock` file will be checked for updates as well.
+Currently support channels from anaconda and prefix.dev, other channels are not supported yet.
 
-Currently only lock file maintenance is supported.
+Whenever the pixi config in `pyproject.toml` or `pixi.toml` file is updated, `pixi.lock` file will be checked for updates as well.
+
+Renovatebot will pick a pixi version based on known mapping between lockfile version and pixi version.
+This is done in order to avoid upgrading the lock file version.
+Optionally you can set `constraints.pixi` in your config to override this.
+
+The minimal support version of pixi is `0.40.0` (`pixi lock` has only been introduced with that version).
+Therefore when you are setting `constraints.pixi` in your config, please be careful to pick a supported pixi version.
+
+This also means if you have a lock file with version `<6`, renovatebot can't generate lock file with same version, it will upgrade lock file to latest version.
+
+If you want to map a mirror repo to official repo, you can use `registryAliases`:
+
+for pixi.toml:
+
+```toml
+[project]
+name = "example-project"
+authors = []
+channels = ["https://mirrors.of.some.company/conda/conda-forge"]
+platforms = ["win-64"]
+version = "0.1.0"
+```
+
+```json
+{
+  "registryAliases": {
+    "https://mirrors.of.some.company/conda/conda-forge": "conda-forge"
```

Comment:
```suggestion
    "mirrors.of.some.company/conda/conda-forge": "conda-forge"
```

Reply:
I'm not sure about this... Is `registryAliases` used outside manager so it should not contain url scheme??

https://github.com/renovatebot/renovate/blob/a4f41929086ae477665e67d4ead6d7f08f6c6185/lib/modules/manager/pixi/extract.ts#L145-L151

Reply:
OK I find pixi has native support for registry aliases, so I'll remove it from here